cli/pcluster/config_sanity.py

```diff
@@ -343,8 +343,10 @@ def validate(self, resource_type, resource_value):  # noqa: C901 FIXME
                         "arn:%s:dynamodb:%s:%s:table/parallelcluster-*" % (partition, self.region, account_id),
                     ),
                     (
-                        ["cloudformation:DescribeStacks"],
-                        ["cloudformation:DescribeStackResource"],
+                        [
+                            "cloudformation:DescribeStacks",
+                            "cloudformation:DescribeStackResource",
```

Comment:
Is the trailing comma allowed here?

Reply:
yes it is. In any case tox reformatted to a single line